bip-0391.mediawiki

```diff
@@ -0,0 +1,98 @@
+<pre>
+  BIP: ???
+  Layer: Applications
+  Title: raw() as subscript in Output Script Descriptors
+  Author: Pieter Wuille pieter@wuille.net
+          Matias Furszyfer mfurszy@protonmail.com
+  Comments-Summary: No comments yet.
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-???
+  Status: Draft
+  Type: Informational
+  Created: 2024-12-16
+  License: BSD-2-Clause
+</pre>
+
+==Abstract==
+
+This document specifies `raw()` as subscript for output script descriptors.
+`raw()` encapsulates a raw hex script. This BIP allows `raw()` to be used in
+the context of other descriptors.
+
+==Copyright==
+
+This BIP is licensed under the BSD 2-clause license.
+
+==Motivation==
+
+Allowing arbitrary hex data to be wrapped in `sh()`, `wsh()`, or even within the `TREE`
+argument of a `tr(KEY, TREE)` descriptor enables the representation of currently
+inexpressible information in the descriptors' language.
+
+Specifically, the absence of this feature limits the representation of non-standard redeem
+and witness scripts. This occurs because they can currently only be represented as top-level
+`raw(HEX)` descriptors, which retain only the output script information and lack the ability
+to preserve the actual script.
+
+Additionally, as noted [https://github.com/bitcoin/bitcoin/issues/24114#issuecomment-1127978154 here],
+there are other useful scenarios for this feature. For example, it allows representing
+in a descriptor that we lack full knowledge of all solvability paths but can still solve
+the output, such as a taproot tree where we know only one of its paths, or even also
+participating in signing a script without knowing all participants' keys, relying solely
+on the script structure.
```

Comment:
This paragraph is hard to parse, perhaps you could split the second sentence into multiple sentences.